src/lib/notion/announcements.ts

```diff
@@ -1,10 +1,13 @@
 import notion from '.';
-import { getPageIds } from './utils';
+import { getPageBySlug, getPageIds } from './utils';
 import { RichTextType } from '@/components/RichText';
 
 export type AnnouncementDataType = {
   announcementPageId: string;
+  slug: string;
   title: string;
+  description: string;
+  homePageURL: string;
```

Comment:
```suggestion
  homePageURL: string; // {* explain that this is link to Cloudinary image to control image that appears on the Home Page carousel (independently of the image shown in the Events page *}
```

* Might have to make this change to Event Cards too